paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -63,8 +71,7 @@ public class AccessibilityRenderExtension : RenderExtension {
     }
   }
 
-  private var unmergedNodes: List<SemanticsNode>? = null
-
+  @SuppressLint("VisibleForTests")
```

Comment:
revert.  i'm guessing this is because of `ViewRootForTest`.  If so, see other comment.